raiden-cli/src/utils/payments.ts

```diff
@@ -1,8 +1,17 @@
 import { RaidenTransfer } from 'raiden-ts';
-import { ApiPayment } from '../types';
+import { ApiPayment, ApiPaymentEvents } from '../types';
+
+function transformTransferDirectionToPaymentEvent(
+  direction: RaidenTransfer['direction'],
+  // @ts-ignore: Function lacks ending return statement and return type does not include 'undefined'
+): ApiPaymentEvents {
+  if (direction === 'sent') return ApiPaymentEvents.sent;
```

Comment:
As there's just 2 possibilities, you could simply use if/else, a conditional expression on return, or possibly (as the `ApiPaymentEvents` is an enum and `direction` is literal), something like `return ApiPaymentEvents[direction];` (not sure if this will work here, but it should with enums+literals.

Reply:
> ApiPaymentEvents[direction];

That looks clean to me. So I can get rid of this whole stupid function at all and code remains slim.

Reply:
:heavy_check_mark: